splash.py

```diff
@@ -6,6 +6,11 @@
 def splash(i, o):
     image = PIL.Image.open("splash.png").convert('L')
     image = invert(image)
-    image = image.convert(o.device.mode)
```

Comment:
The image still has to be shown when output device isn't an emulator device, so this change can't be merged as-is since it'd disable the emulator regardless of that =)